models/dirwatcher.js

```diff
@@ -0,0 +1,33 @@
+import fs from "fs";
+import { csvToJSON } from './../utils';
+import Importer from './importer';
+
+// async - flag, describing whether to emit asynchronous import call or not
+
+export default class DirWatcher {
+  constructor(emitter) {
+    this.emitter = emitter;
+    this.emitter.on("changed", (path, async) => {
```

Comment:
Move this subscription into `Importer` constructor, passing `emitter` object as an argument to it. Than create an instance of that class in *index.js* file, so this class wouldn't know about another one (i.e. separate classes, make no cross imports)